projects/opendr_ws/src/perception/scripts/rgbd_hand_gesture_recognition.py

```diff
@@ -117,15 +120,32 @@ def preprocess(self, image, depth_img):
         return img
```

Comment:
Can't directly comment on unchanged lines, some comments:

- lines 102 and 108 above should be triple double-quotes instead of single-quotes
- docstring has wrong argument names `image_data`, `depth_data` instead of  `image`, `depth_img`